lib/Rx/Observable/ForkJoinObservable.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Rx\Observable;
+
+use Rx\Observable;
+use Rx\Observer\AutoDetachObserver;
+use Rx\Observer\CallbackObserver;
+use Rx\ObserverInterface;
+use Rx\Disposable\CompositeDisposable;
+use Rx\Scheduler\ImmediateScheduler;
+use Rx\SchedulerInterface;
+
+class ForkJoinObservable extends Observable {
+
+    /**
+     * @var Observable[]
+     */
+    private $observables;
+
+    private $values = [];
+
+    private $completed = 0;
+
+    private $resultSelector;
+
+    public function __construct(array $observables = [], callable $resultSelector = null) {
```

Comment:
I checked the original RxJS 4 and RxJS 5 tests and both test forkJoin with no arguments so it should probably be both optional.

https://github.com/ReactiveX/rxjs/blob/master/spec/observables/forkJoin-spec.ts#L192  
https://github.com/Reactive-Extensions/RxJS/blob/master/tests/observable/forkjoin.js#L127